debian/systemd.install

```diff
@@ -17,6 +17,7 @@ lib/udev/rules.d/71-seat.rules
 lib/udev/rules.d/99-systemd.rules
 usr/bin/systemd-cgls
 usr/bin/systemd-cgtop
+usr/bin/systemd-coredumpctl
```

Comment:
Oh, `systemd-coredumpctl` is also in `GENERAL_ALIASES`, which will make a symlink, so I think you're right here.